### ISSUE TITLE
SF Powerkit reconcile is removing permissions related to Task and Events and User Permission

### ISSUE THREAD

**Issue by RajaPaul** [dxatscale/sfpowerkit#675]
**Describe the bug**
Task and Event records take the fields from the Activity Object, but when we are doing a reconcile of the profile, the permissions related to Task and Event are getting removed.

Also, towards the end of a Custom Profile where we have userPermissions, the reconcile command is removing these userPermissions. which it should not

**To Reproduce**

1. sfdx sfpowerkit:source:profile:retrieve -n "CUSTOM_PROFILE"
2. sfdx sfpowerkit:source:profile:reconcile -s -n "CUSTOM_PROFILE" --folder .\force-app\

**Expected behavior**
1. User Permissions Should Not get removed
2. Permissions related to Tasks and Events should stay, if activity custom fields are present in repository

**Screenshots**
N/A

**Desktop (please complete the following information):**

-   OS: Windows
-   Version of sfpowerkit: This is present in many version, i got it fixed in a past version, but we are using 3.0.0 and the latest
-   Salesforce Org Version summer 22

**Additional context**

**Comment by sumanthreddybolledla** [comment]
Hi @RajaPaul, you mentioned you are using 3.0.0 version. Could you update your sfpowerkit to 4.2.8 which is the latest version and try?

**Comment by Caitlyn-Mills** [comment]
> Hi @RajaPaul, you mentioned you are using 3.0.0 version. Could you update your sfpowerkit to 4.2.8 which is the latest version and try?

Yep @RajaPaul if you can try version 4.2.8 as recommended. Thanks

**Comment by Caitlyn-Mills** [comment]
Closing this issue due to inactivity. Please reopen if the issue is persisting.

**Comment by DougMidgley** [comment]
This issue is also affecting us v 4.2.11. Any guidance on that @Caitlyn-Mills?

**Comment by Caitlyn-Mills** [comment]
@DougMidgley reopening this issue. 
@genoud would you be able to look into this one?

**Comment by pinkj24** [comment]
This issue is also affecting v5.0.0. Is there an update on the investigation on this please?

**Comment by joernflath** [comment]
This bug affects us as well: UserPermissions are being removed during reconcile
